src/main/java/com/services/OrganizationService.java

```
@@ -78,10 +84,11 @@ public void deleteOrg (int id){
             throwable.printStackTrace();
```

This is the place you should throw the not found I guess.

---

Catch Class not found together with SQL Exception